### PR TITLE
Report channel fix

### PR DESCRIPTION
**Pull request by ggabernet**
- Adds the report Rmarkdown to a channel instead of reading directly from assets, to fix errors running on AWS

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](<https://github.com/>nf-core/bcellmagic/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/bcellmagic _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: bcellmagic.nf [hunk 1]
```diff
@@ -75,9 +75,16 @@ if( params.imgtdb_base ){
 /* --          CONFIG FILES                    -- */
 ////////////////////////////////////////////////////
 
-ch_multiqc_config        = file("$projectDir/assets/multiqc_config.yaml", checkIfExists: true)
+ch_multiqc_config        = channel.fromPath("$projectDir/assets/multiqc_config.yaml", checkIfExists: true)
```
Comment (apeltzer) [inline bcellmagic.nf:78]:
```suggestion
ch_multiqc_config        = Channel.fromPath("$projectDir/assets/multiqc_config.yaml", checkIfExists: true)
```

File: bcellmagic.nf [hunk 2]
```diff
@@ -75,9 +75,16 @@ if( params.imgtdb_base ){
...
 ch_multiqc_custom_config = params.multiqc_config ? Channel.fromPath(params.multiqc_config) : Channel.empty()
 
+// Rmarkdown report file
+ch_rmarkdown_report      = channel.fromPath( ["$projectDir/assets/repertoire_comparison.Rmd",
```
Comment (apeltzer) [inline bcellmagic.nf:82]:
```suggestion
ch_rmarkdown_report      = Channel.fromPath( ["$projectDir/assets/repertoire_comparison.Rmd",
```

**Review by apeltzer — APPROVED** [review]
I think we usually have `Channel` instead of `channel` - otherwise looking good 👍🏻

**Comment by ggabernet** [comment]
Thanks, unfortunately I'll have to do the template update so the tests pass first, but hopefully will be done with it tomorrow. PR coming!

**Pull request closed without merging**